common/src/main/java/com/microsoft/hydralab/performance/PerformanceInspectorManagementService.java

```diff
@@ -0,0 +1,86 @@
+package com.microsoft.hydralab.performance;
+
+import com.microsoft.hydralab.performance.impl.*;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.*;
+
+@Service
+public class PerformanceInspectorManagementService implements IPerformanceInspectionService {
+    static final ScheduledExecutorService timerExecutor = Executors.newScheduledThreadPool(5 /* corePoolSize */);
```

Comment:
Can the type of inspectPerformanceTimerList also be `Map<String, ScheduledFuture<?>>`?